author.rb

```diff
@@ -0,0 +1,12 @@
+class Author
+  private
+
+  attr_accessor id, items
```

Comment:
maybe you should put : before each parameter, something like this:

`attr_Accessor :id, :items`

Reply:
Oh you're right @rbreva thank you for letting me know, I'll send an update for tomorrow 🙌🏽